src/hmm/hmm-utils.cc

```diff
@@ -1289,5 +1289,19 @@ void ChangeReorderingOfAlignment(const TransitionModel &trans_model,
   }
 }
 
+void PdfToPhonesDict(const TransitionModel &trans_model,
```

Comment:
sorry.. please rename to GetPdfToPhonesMap().  You should probably put this in transition-model.{h,cc}.
And since the pdf-ids are numbered from 0 to [num-pdfs - 1], you don't need a map, it can be a
vector.  (but keep the Map() in the name).